Formula/gz-harmonic.rb

```diff
@@ -0,0 +1,55 @@
+class GzHarmonic < Formula
+  include Language::Python::Virtualenv
+
+  desc "Collection of gazebo simulation software"
+  homepage "https://github.com/gazebosim/gz-harmonic"
+  url "https://github.com/gazebosim/gz-harmonic.git", branch: "main"
+  version "0.999.999~0~20230728"
+  license "Apache-2.0"
+
+  head "https://github.com/gazebosim/gz-harmonic.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.11" => [:build, :test]
+
+  depends_on "gz-cmake3"
+  depends_on "gz-common5"
+  depends_on "gz-fuel-tools9"
+  depends_on "gz-gui8"
+  depends_on "gz-launch7"
+  depends_on "gz-math7"
+  depends_on "gz-msgs10"
+  depends_on "gz-physics7"
+  depends_on "gz-plugin2"
+  depends_on "gz-rendering8"
+  depends_on "gz-sensors8"
+  depends_on "gz-sim8"
+  depends_on "gz-tools2"
+  depends_on "gz-transport13"
+  depends_on "gz-utils2"
+  depends_on macos: :mojave # c++17
```

Comment:
nit: this line can be removed

Reply:
https://github.com/osrf/homebrew-simulation/pull/2326/commits/7c5e58ce991012f16f92b61c43890c797c8a8323